stratum/hal/bin/barefoot/docker/start-stratum-container.sh

```diff
@@ -17,7 +17,12 @@ elif [[ -z "$PLATFORM" ]]; then
 fi
 
 # Set Docker network options.
-if [[ "$PLATFORM" == 'barefoot-tofino-model' ]]; then
+# On tofino-model or switches with BSP support, we bind to the host network
+# directly. The BSP on Wedge devices needs access to the usb0 interface.
+if [[ "$PLATFORM" == 'barefoot-tofino-model' ]] || \
+   [[ "$PLATFORM" == "x86-64-accton-wedge100bf-32x-r0" ]] || \
+   [[ "$PLATFORM" == "x86-64-accton-wedge100bf-32qs-r0" ]] || \
+   [[ "$PLATFORM" == "x86-64-accton-wedge100bf-65x-r0" ]]; then
```

Comment:
@bocon13 @Yi-Tseng

I found no other way yet. For reference: the BSP will do a http request against the BMC in `bf_pltfm_bd_type_init`.

If this is our solution, we have to update the run README.

Reply:
I don't see a big problem with this.

If we don't want to use the host network, we could consider moving the usb0 interface into the container's netns. (Downside is that the host wouldn't be able to access the BMC.)

Or, we could create a bridge... But that's definitely going to be more complicated.

Reply:
Sounds good.

I looked at the `README.run.md`, but all mentions of `--network` apply to tofino model.